dener8/programmers_기능개발.py

```diff
@@ -0,0 +1,29 @@
+'''
+인자 두 개 모두 max_len = 100
+ex. 95% -> 99%(1일) -> 104%(2일) : 2일 걸리는 경우
+
+return은 한 묶음으로 배포되는 개수를 쌓으면 됨
+'''
+from collections import deque
+
+def solution(progresses, speeds):
+    answer = []
+    progresses = deque(progresses)
+    speeds = deque(speeds)
+
+    while len(progresses) != 0 and len(speeds) != 0:
```

Comment:
```suggestion
    while not(len(progresses) == 0 or len(speeds) == 0):
```

이거 이렇게 하는게 더 생각하기 편하지 않음?